apps/system/js/app_install_manager.js

```diff
@@ -242,7 +243,9 @@
 
       if (manifest.size) {
         this.size.removeAttribute('data-l10n-id');
-        this.size.textContent = this.humanizeSize(manifest.size);
+        this.humanizeSize(manifest.size).then(val => {
+          this.size.textContent = val;
+        });
```

Comment:
Wouldn't it be a better approach to pass this through l20n.js?  In the future we'd like to get rid of `humanizeSize` altogether and replace it with in-localization macros/filters.